easybuild/easyconfigs/b/Boost/Boost-1.74.0-gompi-2020b.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.74.0'
+
+homepage = 'https://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://dl.bintray.com/boostorg/release/%(version)s/source/']
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['afff36d392885120bcac079148c177d1f6f7730ec3d47233aa51b0afa4db94a5']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.5'),
+]
+
+configopts = '--without-libraries=python'
+
+# also build boost_mpi
+boost_mpi = True
```

Comment:
-

Reply:
`*`?

Reply:
That was a - sign when i wrote it. but, perhaps it should just be set to False. The easyblock kind of gives us to many ways to achieve the same thing i think.

Reply:
I agree, the functionality of `boost_mpi` can easily be moved to the toolchainopt `usempi`, but that means updating all easyconfigs as well